src/patternfly/components/Form/form.scss

```diff
@@ -62,123 +63,120 @@ $pf-v5-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "l
 
   // Group control
-  --#{$form}__group-control--m-inline--child--MarginRight: var(--#{$pf-global}--spacer--lg);
-  --#{$form}__group-control__helper-text--MarginBottom: var(--#{$pf-global}--spacer--xs);
-  --#{$form}__group-control--m-stack--Gap: var(--#{$pf-global}--spacer--sm);
+  --#{$form}__group-control--m-inline--child--MarginRight: var(--pf-t--global--spacer--lg);
+  --#{$form}__group-control__helper-text--MarginBottom: var(--pf-t--global--spacer--xs);
+  --#{$form}__group-control--Gap: var(--pf-t--global--spacer--sm);
+  --#{$form}__group-control--m-stack--Gap: var(--pf-t--global--spacer--sm);
   --#{$form}__group-control--m-stack__helper-text--MarginTop: calc(var(--#{$form}__group-control--m-stack--Gap) * -1 + var(--#{$form}__helper-text--MarginTop--base));
 
   // Actions
-  --#{$form}__actions--child--MarginTop: var(--#{$pf-global}--spacer--sm);
-  --#{$form}__actions--child--MarginRight: var(--#{$pf-global}--spacer--sm);
-  --#{$form}__actions--child--MarginBottom: var(--#{$pf-global}--spacer--sm);
-  --#{$form}__actions--child--MarginLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$form}__actions--child--MarginTop: var(--pf-t--global--spacer--sm);
+  --#{$form}__actions--child--MarginRight: var(--pf-t--global--spacer--sm);
+  --#{$form}__actions--child--MarginBottom: var(--pf-t--global--spacer--sm);
+  --#{$form}__actions--child--MarginLeft: var(--pf-t--global--spacer--sm);
   --#{$form}__actions--MarginTop: calc(var(--#{$form}__actions--child--MarginTop) * -1);
   --#{$form}__actions--MarginRight: calc(var(--#{$form}__actions--child--MarginRight) * -1);
   --#{$form}__actions--MarginBottom: calc(var(--#{$form}__actions--child--MarginBottom) * -1);
   --#{$form}__actions--MarginLeft: calc(var(--#{$form}__actions--child--MarginLeft) * -1);
 
   // Helper text
-  --#{$form}__helper-text--MarginTop--base: var(--#{$pf-global}--spacer--xs);
+  --#{$form}__helper-text--MarginTop--base: var(--pf-t--global--spacer--xs);
   --#{$form}__helper-text--MarginTop: var(--#{$form}__helper-text--MarginTop--base);
-  --#{$form}__helper-text--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$form}__helper-text--Color: var(--#{$pf-global}--Color--100);
+  --#{$form}__helper-text--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$form}__helper-text--Color: var(--pf-t--global--text--color--regular);
 
-  // here
   // Helper text icon
-  --#{$form}__helper-text-icon--FontSize: var(--#{$pf-global}--FontSize--md);
-  --#{$form}__helper-text-icon--MarginRight: var(--#{$pf-global}--spacer--xs);
-
-  // States
-  --#{$form}__helper-text--m-success--Color: var(--#{$pf-global}--success-color--200);
-  --#{$form}__helper-text--m-warning--Color: var(--#{$pf-global}--warning-color--200);
-  --#{$form}__helper-text--m-error--Color: var(--#{$pf-global}--danger-color--100);
+  --#{$form}__helper-text-icon--FontSize: var(--pf-t--global--font--size--body--md);
+  --#{$form}__helper-text-icon--MarginRight: var(--pf-t--global--spacer--xs);
```

Comment:
Since we're now using the HelperText components, it doesn't look like we would need these vars or the __helper-text-icon declaration block